packages/ag-charts-enterprise/src/features/error-bar/errorBar.ts

```diff
@@ -88,14 +99,52 @@ export class ErrorBars
         this.cartesianSeries = ctx.series as _ModuleSupport.CartesianSeries<any, any>;
         const { contentGroup } = this.cartesianSeries;
 
-        this.groupNode = new _Scene.Group({ name: `${contentGroup.id}-series-errorBars` });
+        this.groupNode = new _Scene.Group({
+            name: `${contentGroup.id}-series-errorBars`,
+            zIndex: _ModuleSupport.Layers.SERIES_ERRORBAR_ZINDEX,
+            zIndexSubOrder: this.cartesianSeries.getGroupZIndexSubOrder('error-bars'),
+        });
         contentGroup.appendChild(this.groupNode);
         this.selection = _Scene.Selection.select(this.groupNode, () => this.errorBarFactory());
 
-        this.destroyFns.push(this.cartesianSeries.addListener('data-model', (event) => this.onDataProcessed(event)));
+        this.destroyFns.push(
+            this.cartesianSeries.addListener(
+                'processData-prerequest',
+                (event: _ModuleSupport.SeriesPrerequestDataEvent) => this.onPrerequestData(event)
+            )
+        );
+        this.destroyFns.push(
+            this.cartesianSeries.addListener('data-update', (event: _ModuleSupport.SeriesDataUpdateEvent) =>
+                this.onDataUpdate(event)
+            )
+        );
+        this.destroyFns.push(
+            this.cartesianSeries.addListener('visibility-changed', (event: _ModuleSupport.SeriesVisibilityEvent) =>
+                this.onToggleSeriesItem(event)
+            )
+        );
```

Comment:
Nit: Maybe just condense this to a single `.push()` call with multiple arguments?

Reply:
Done.